src/searchengine/nova/helpers.py

```diff
@@ -1,4 +1,4 @@
-#VERSION: 1.43
+#VERSION: 1.45
```

Comment:
1.44 and in other file.

Reply:
I just aligned the 2 helpers versions

Reply:
Please check again, there is no need to skip over `1.44` and directly to `1.45`